main.ts

```diff
@@ -286,6 +287,16 @@ class CustomConstructsStack extends TerraformStack {
       }
       if (languages.includes("go")) {
         secrets.forGo(repo.resource, githubProvider);
+
+        // repo to publish go packages to
+        new GithubRepository(this, `${repoName}-go`, {
+          description: `CDK for Terraform Go bindings for ${repoName}.`,
+          topics,
```

Comment:
Is there a way we can refactor `topics` to not always add topics like `prebuilt-provider` as currently planned?
<img width="454" alt="image" src="https://user-images.githubusercontent.com/1112056/200574631-0c5ffaa0-654a-48d8-b77e-57d415b84152.png">

Reply:
Ah good catch!

Reply:
Should be solved now